### ISSUE TITLE
Exception after updated to Angular "2.0.0-rc.2"

### ISSUE THREAD

**Issue by rameshbas** [michaelbromley/ng2-pagination#50]
I updated to Angular "2.0.0-rc.2", from an earlier beta version  "2.0.0-beta.13", I am getting errors in pagination 

```
EXCEPTION: Error in app/users/user-list.component.html:37:13 angular2.dev.js:25644
EXCEPTION: Error in app/users/user-list.component.html:37:13BrowserDomAdapter.logError @ angular2.dev.js:25644 angular2.dev.js:25644 
ORIGINAL EXCEPTION: TypeError: Cannot read property 'totalItems' of undefinedBrowserDomAdapter.logError @ angular2.dev.js:25644 angular2.dev.js:25644 
ORIGINAL STACKTRACE:BrowserDomAdapter.logError @ angular2.dev.js:25644 angular2.dev.js:25644 TypeError: Cannot read property 'totalItems' of undefined     at PaginatePipe.transform (paginate.pipe.ts:42)     at AppView._View_UserListComponent0.detectChangesInternal (UserListComponent.template.js:527)     at AppView.detectChanges (angular2.dev.js:23638) 
```

I noticed some compiler error too:

**Comment by michaelbromley** [comment]
Hi.

First off - you put your entire issue text in the title. The title should be a short description (like a sentence) of the problem. The rest of the text should go in the body of the issue. I edited that now so take a look how I did it for future reference.

As to the issue itself, can you show me your component's template and controller code?

**Comment by rameshbas** [comment]
Mike,Any update on this?  I need to use Alpha release of angular2.
ThanksRamesh Bas

**Comment by michaelbromley** [comment]
Yes - I am waiting for you to provide a bit more information:

> can you show me your component's template and controller code?

**Comment by rameshbas** [comment]
Mike,I have attached a simple program. The same works in the earlier beta version.Thanks for your helpRamesh Bas

**Comment by michaelbromley** [comment]
Hi,

You cannot just send an email attachment to GitHub and expect it to work. Also, even if it did, I do not have time to read though a large number of files to figure things out I'm afraid.

That is why in the issue guidelines, I ask you to provide relevant code and reproduce the issue. Otherwise I simply won't have time to help you.

**Comment by michaelbromley** [comment]
Whatever this issue was should be fixed in 0.3.5
